.goreleaser.yaml

```diff
@@ -107,6 +107,34 @@ docker_signs:
     - '${artifact}'
     - "--yes" # needed on cosign 2.0.0+
 
+brews:
+  - tap:
+      owner: openfga
+      name: homebrew-tap
+      pull_request:
+        enabled: true
+    homepage: "https://openfga.dev/"
+    description: "A high performance and flexible authorization/permission engine built for developers and inspired by Google Zanzibar."
```

Comment:
Description here should probably be something about the CLI rather than about OpenFGA. People shouldn't accidentally assume they are installing the server

Reply:
```suggestion
    description: "A cross-platform CLI to interact with an OpenFGA server."
```